.erb/configs/.eslintrc

```diff
@@ -2,6 +2,8 @@
   "rules": {
     "no-console": "off",
     "global-require": "off",
-    "import/no-dynamic-require": "off"
+    "import/no-dynamic-require": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/ban-ts-comment": "off"
```

Comment:
Why this rule? Seems like there aren't any ts bans in this PR

Reply:
There is a `@ts-ignore` in [.erb/configs/webpack.config.renderer.dev.ts](https://github.com/electron-react-boilerplate/electron-react-boilerplate/pull/3165/files/8c3be67f48677c37232fc8aa86c00668ac132905#diff-98d366b10a60c9624cca34f2d0e8bc99b2e130d73fa9aadca5bf00631733fe73)